innovation.game.php

```diff
@@ -11727,8 +11836,38 @@ function stInterInteractionStep() {
                         }
                     }
                     break;
-                }
 
+                // id 155, Artifacts age 5: Boerhavve Silver Microscope
+                case "155N1B":
+                    // "Draw and score a card of value equal to the sum of the values of the cards returned."
+                    $first_age = self::getGameStateValue('auxiliary_value');
+                    $second_age = self::getGameStateValue('age_last_selected');
+                    self::executeDraw($player_id, $first_age + $second_age, 'score');
```

Comment:
Don't we only want to draw if we know that two cards were actually returned?

Maybe check that `$first_age > 0` and `$second_age > 0`?

Reply:
no card returned would be 0.

Reply:
if it said, "if you do" then it would matter if two cards are returned.

Reply:
Hmm let's think, so the card says "Draw and score a card of value equal to the sum of the values of the cards returned". I guess that means if you only return one card, then you draw and score a card of that same value? And if they return two cards then you take the sum?

Please test that both cases work.

Reply:
right
I've tested both cases including an empty hand.